openedx/openedx-platform
# Re-enable content libraries runtime tests [FC-0062]

## Description

Somewhere in the series of refactorings that this app has undergone, it seems that we removed TestCase from the inheritance tree of the tests in `openedx/core/djangoapps/content_libraries/tests/test_runtime.py`, effectively skipping the test cases.

This PR re-enables the tests and fixes what's broken since they were disabled.

## Supporting information

Builds on/replaces: https://github.com/openedx/edx-platform/pull/35531
Private-ref: [FAL-3927](https://tasks.opencraft.com/browse/FAL-3927)

## Testing instructions

Check the logs of `openedx-1-with-lms` and `openedx-2-with-cms` to confirm that tests are run for `openedx/core/djangoapps/content_libraries/tests/test_runtime.py`. Some will be skipped for the lms, some skipped for the cms, but all tests should be run by one or the other.

## Deadline

ASAP -- want to get into Sumac
